### PR TITLE
Add AR 6.x compatibility

### PR DESCRIPTION
**Pull request by bquorning**
Closes #15

### CODE REVIEW

File: Appraisals [hunk 1]
```diff
@@ -5,4 +5,15 @@ end
...
+
+appraise "activerecord_6.0" do
+  gem "activerecord", "~> 6.0.0"
+  gem "sqlite3", "~> 1.4"
+end
+
+appraise "activerecord_6.1" do
+  gem "activerecord", "~> 6.1.0"
+  gem "sqlite3", "~> 1.4"
 end
```
Comment (grosser) [inline Appraisals:19]: needs are rebase ?

**Review by grosser — APPROVED** [review]

File: Appraisals (hunk 1 above)
Comment (bquorning) [reply Appraisals]: It's ready.

File: Gemfile.lock [hunk 2]
```diff
@@ -2,7 +2,7 @@ PATH
   remote: .
   specs:
     active_record-comments (0.1.4)
-      activerecord (>= 4, < 6)
+      activerecord (>= 4, < 7)
```
Comment (grosser) [inline Gemfile.lock:5]: <6.2 is safer though

**Comment by bquorning** [comment]
Ready for another review.

**Comment by grosser** [comment]
ci failed

**Comment by bquorning** [comment]
CI switched _all the builds_ from green to red just as I posted my comment. Stupid JS UI 🤷🏼 

Anyway, I pushed a fix.

**Comment by grosser** [comment]
0.2.0

**Comment by dadah89** [comment]
Big thanks for adding/releasing AR 6 compatibility, really appreciated

**Pull request merged**